src/layout/tooltip/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export default function Tooltip() {
+
+    const style = {
+        position: "absolute",
+        left: "0px",
+        top: "0px",
+        visibility: "hidden",
+        padding: "2px",
```

Comment:
```suggestion
        padding: props.padding ? `${props.padding}px` : '2px',
```

Using the padding prop and having a default state otherwise, again gives flexibility to using the component. Optionally a similar solution can be implemented for the `left` and `top` style attributes too.